### ISSUE TITLE
Change Endianness (question)

### ISSUE THREAD

**Issue by bucweat** [jDataView/jBinary#64]
I’m parsing PCAP files...PCAP headers are little endian however Ethernet/IP/TCP and payload in most cases are big endian. Is it possible to switch endianness while parsing a given type in a typeset?

**Comment by RReverser** [comment]
Yes, absolutely, this is covered in docs as far as I remember.

**Comment by bucweat** [comment]
Assuming you are talking about the wiki. Endianness is mentioned when defining typeset. Can this be used in a template?

**Comment by RReverser** [comment]
I don't think it is, but note that typeset config just changes the default endianness; instead, you can pass it on field-by-field basis as type param by using `['uint16', true]` for explicit little-endian and false for big-endian.

**Comment by bucweat** [comment]
Ah ha!!!!! That is exactly what I was looking for! Thank you :-)
